src/client/runner/test-context-storage.js

```diff
@@ -0,0 +1,44 @@
+import hammerhead from './deps/hammerhead';
+
+var JSON = hammerhead.json;
+
+const STORAGE_KEY_PREFIX = "testCafeRunner|testContextStorage|";
```

Comment:
a suppose we can use just `runner|` as a prefix. For example we use the `automation|` prefix in automations